controllers/comments.controller.js

```diff
@@ -19,4 +19,11 @@ exports.postCommentByArticleId = (req, res, next) => {
         if (err.code === '23503') res.status(404).send({message: 'user not found'})
         next(err)
     })
+}
+
+exports.deleteCommentById = (req, res, next) => {
+    const {comment_id: id} = req.params
+    removeCommentById(id)
+    .then(()=>{res.status(204).send()})
```

Comment:
When not sending back a body we can opt to use res.sendStatus() instead, the way you've done it is totally fine too though